test/Microsoft.AspNetCore.Mvc.Test/MvcServiceCollectionExtensionsTest.cs

```diff
@@ -226,6 +228,22 @@ public void AddMvcCore_ReusesExistingApplicationPartManagerInstance_IfFoundOnSer
             Assert.Same(manager, descriptor.ImplementationInstance);
         }
 
+        [Fact]
+        public void AddMvcCore_CheckAdded_MvcJsonOption()
+        {
+            var services = new ServiceCollection();
+           
+            services.AddMvcCore().
+                AddJsonOptions((options) =>
+            {
+                options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+                options.SerializerSettings.Formatting = Formatting.Indented;
+            });
+
+            // Assert
+            var descriptor = Assert.Single(services, d => d.ServiceType == typeof(IConfigureOptions<MvcJsonOptions>));
```

Comment:
I can make this change as part of merging this - you don't need to assign this to a variable if you don't intend on doing anything with it. I'm surprised the compiler doesn't warn here.